scripts/qa_crowbarsetup.sh

```diff
@@ -1265,14 +1270,16 @@ function onadmin_allocate()
     done
     echo "Sleeping 50 more seconds..."
     sleep 50
+    local controllernodes=(
+            $(get_all_discovered_nodes | head -n 2)
+        )
     echo "Setting first node to controller..."
-    local controllernode=$(get_all_discovered_nodes | head -n 1)
-    local t=$(mktemp).json
+    set_node_role_and_platform ${controllernodes[0]} "controller" "suse-11.3"
 
-    knife node show -F json $controllernode > $t
-    json-edit $t -a normal.crowbar_wall.intended_role -v "controller"
-    knife node from file $t
-    rm -f $t
+    if [ -n "$want_sles12_controller" ] ; then
+        echo "Setting second node as SLE12 controller ..."
+        set_node_role_and_platform ${controllernodes[1]} "no_role" "suse-12.0"
```

Comment:
I'm sorry but until we will not finish adapting services for SLE12 controller, we will have problem with double controllers with different OS SLE11SP3 and SLE12. This is because during creating proposal node with controller role is assigned and mostly will be taken this with SLE12 on board and some barclamps still have sles-12.0 platfrom excluded.